react-carousel-component/src/carousel.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState } from 'react';
+
+export default function Carousel({ images }) {
+  const [curImage, setCurImage] = useState(0);
+  const length = images.length;
+
+  function nextSlide() {
+    setCurImage(curImage === length - 1 ? 0 : curImage + 1);
+  }
+
+  function prevSlide() {
+    setCurImage(curImage === 0 ? length - 1 : curImage - 1);
+  }
+
+  useEffect(() => {
```

Comment:
Excellent use of `useEffect`. I guess you won't need the next exercise! ;-)

Well, almost. You need to add dependencies. I guess you'll need the next exercise, after all!